go/service/kvstore.go

```diff
@@ -44,21 +44,34 @@ func (h *KVStoreHandler) assertLoggedIn(ctx context.Context) error {
 	return nil
 }
 
+func rewriteMisleadingError(err error) error {
```

Comment:
Inline this. Cause it's very specific and called in once place. Easier to read inline